### PR TITLE
refactor (AABlocks): summarized LampColor in register

### PR DESCRIPTION
**Pull request by JCrisses**

### CODE REVIEW

**Comment by Ellpeck** [comment]
Please have those formatting changes fixed before merging, all those unnecessary newlines are horrible

File: src/main/java/de/ellpeck/actuallyadditions/mod/item/AAItems.java [hunk 1]
```diff
@@ -14,16 +14,14 @@
 public class AAItems {
 
   public static final Item BLACK_QUARTZ = null;
-  public static final Item PALIS_CRYSTAL = null;
```
Comment (Ellpeck) [inline src/main/java/de/ellpeck/actuallyadditions/mod/item/AAItems.java]: why did you remove this 🤔

**Comment by Shadows-of-Fire** [comment]
well, this is a disaster.

**Comment by FanamyWirato** [comment]
Sorry guys, the merge Request was accidental - our work is not completed yet.

**Comment by Shadows-of-Fire** [comment]
I would prefer the registries stay as they are, there's no reason to merge that data into the enum
